.travis.yml

```diff
@@ -1,9 +1,19 @@
 sudo: false
 language: erlang
 otp_release:
-- 18.1
-- 19.1
+    - 19.1
+    - 19.2
+    - 19.3
+before_install:
+    - wget https://github.com/rebar/rebar/wiki/rebar
+    - chmod u+x ./rebar
+    - wget https://s3.amazonaws.com/rebar3/rebar3
+    - chmod u+x ./rebar3
+env:
```

Comment:
I think rebar3 only CI test are sufficient, please sync this with gtplib's, see https://github.com/travelping/gtplib/commit/5ed158eda5b96c8c4913f1f9db7b4c6e7eb165b0 (.travis.yml, rebar.config.* and README.md) in one cset, drop the rebar2 support, and make the minimal version R19

And make this a separate PR to we can apply it independent of the diameter changes.